pkg/sentry/memmap/memmap.go

```diff
@@ -254,6 +254,9 @@ type MappingIdentity interface {
 	// exclusive, hence mr.End-1.) It is defined rather than Fsync so that
 	// implementors don't need to depend on the fs package for fs.SyncType.
 	Msync(ctx context.Context, mr MappableRange) error
+
+	// DenyWrite is used to inc/dec inode's counter, WriteCount.
```

Comment:
Instead of one method that takes a bool, why not two methods?  `DenyWrite()` and `AllowWrite()` ?  That makes the code much easier to read.